### PR TITLE
Add backward-cpp

### PR DESCRIPTION
**Pull request by edisongustavo**

### CODE REVIEW

File: recipes/backward-cpp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [not linux]
+
+requirements:
+  build:
+    - cmake
+    - libgcc
```
Comment (ocefpaf) [inline recipes/backward-cpp/meta.yaml]: If you are not using conda's gcc you probably don't need libgcc here.

File: recipes/backward-cpp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - elfutils  # so that the tests compiled in the build step can link to libdw
+
+test:
+  requires:
+    - cmake
+    - gcc
+    - libgcc
```
Comment (ocefpaf) [inline recipes/backward-cpp/meta.yaml]: gcc installs libgcc so you don't need that here.

**Comment by edisongustavo** [comment]
@ocefpaf the bulid is passing, whenever you're ready! :)

**Comment by ocefpaf** [comment]
Hehe Antony was faster. I have limited connection here at UFSC.

PS: due to the Travis-CI outage it may take a while for the feedstock to be
created.

**Pull request merged**